doc/source/release-notes/Percona-Server-5.6.26-74.0.rst

```diff
@@ -0,0 +1,72 @@
+
+ Table API was using stale reference of table to check if table is corrupted. Bug fixed :bug:`1433197`.
+
+ Fixed upstream bug :mysqlbug:`77011` which caused a hanging ``System Lock`` when executing ``FLUSH TABLE ... FOR EXPORT``. Bug fixed :bug:`1454441`.
+
+ Regression introduced by fixing bug :bug:`1438990` in |Percona Server| :rn:`5.6.24-72.2` could lead to slave crash with malformed binary log event. Bug fixed :bug:`1464468`.
+
+ ``FLUSH INDEX_STATISTICS`` / ``FLUSH CHANGED_PAGE_BITMAPS`` and ``FLUSH USER_STATISTICS`` / ``RESET CHANGE_PAGE_BITMAPS`` pairs of commands were inadvertently joined, i.e. issuing either command had the effect of both. The first pair, besides flushing both index statistics and changed page bitmaps, had the effect of ``FLUSH INDEX_STATISTICS`` requiring ``SUPER`` instead of ``RELOAD`` privilege. The second pair resulted in ``FLUSH USER_STATISTICS`` destroying changed page bitmaps. Bug fixed :bug:`1472251`.
+
+ Enabling variable:`super_read_only` with :variable:`read_only` simultaneously would lead to server crash when it was restarted. Bug fixed :bug:`1483956` (Fix was ported from Facebook patch `#14d5d9 <https://github.com/percona/percona-server/commit/14d5d9a94f2cac1ae67afd8a806a0ed581530f7e>`_).
+
+ If a new connection thread was created while a ``SHOW PROCESSLIST`` command or a :table:`INFORMATION_SCHEMA.PROCESSLIST` query was in progress, it could have a negative TIME_MS value returned in the PROCESSLIST output. Bug fixed :bug:`1379582`.
+
+ Enabling :variable:`super_read_only` as a command line option would not enable :variable:`read_only`. Bug fixed :bug:`1389935` (Fix was ported from Facebook patch `#14d5d9 <https://github.com/percona/percona-server/commit/14d5d9a94f2cac1ae67afd8a806a0ed581530f7e>`_).
+
+ Running ``INSERT`` queries with :ref:`innodb_fake_changes_page` enabled could lead to server assertion with ``unknown error code 1000``. Bug fixed :bug:`1410410`.
+
+ ``ALTER TABLE`` did not allow to change ``NULL/NOT NULL`` if foreign key exists. Bug fixed :bug:`1470677` (upstream :mysqlbug:`77591`).
+
+ ``DROP TABLE IF EXISTS`` could break replication if slave had replication filters. Bug fixed :bug:`1475107` (upstream :mysqlbug:`77684`).
+
+ Enabling :ref:`log_archiving` when :option:`--innodb-read-only` option was enabled would cause server to crash. Bug fixed :bug:`1484432`.
+
+ LRU manager thread flushing was being accounted to ``buffer_flush_background`` *InnoDB*  metrics which was wrong and redundant. Bug fixed :bug:`1491420`.
+
+ Fixed a typo in the cleaner thread loop where ``n_flushed`` is added to, instead of reset, by the idle server flushing. This may cause a cleaner thread sleep skip on a non-idle server. Bug fixed :bug:`1491435`.
+
+Other bugs fixed: :bug:`1370002`, :bug:`1287299`, :bug:`1372219`, :bug:`1472256`, :tokubug:`867`.
```

Comment:
1372219 needs a description (and then group the fake changes bugs together in RNs)